pages/common/isisdl.md

```diff
@@ -0,0 +1,28 @@
+# isisdl
+
+> A downloading utility for ISIS of TU-Berlin. Download all your files and videos from ISIS.
+> More information: <https://github.com/Emily3403/isisdl>.
+
+- Start the synchronization process:
+
+`isisdl`
+
+- Limit the download rate to "20 MiB/s" and download with "5" threads:
```

Comment:
```suggestion
- Limit the download rate to 20 MiB/s and download with 5 threads:
```

Reply:
@mfrw suggested this [edit](https://github.com/tldr-pages/tldr/pull/8134#discussion_r899672234). I'm not quite sure if the "correct" way is to quote it or not.

Reply:
:( apologies…. My bad